imports/lib/models/Base.ts

```diff
@@ -64,6 +64,14 @@ class Base<T extends BaseType> extends Mongo.Collection<T> {
     return super.insert(<any>doc, callback);
   }
 
+  // @ts-expect-error See above
+  insertAsync<U>(
+    doc: ValidateShape<U, Partial<T>>,
+    callback?: (err?: Error, id?: string) => void,
+  ): Promise<string> {
+    return super.insertAsync(<any>doc, callback);
```

Comment:
I find it odd that `insertAsync` still takes a callback parameter?  Wouldn't we just `reject` the promise instead?

Reply:
I admittedly didn't think very hard about this, but was just copying what the upstream type declarations did. But it does seem kind of odd. I'll switch